linux/utils/install_uninstall_package.yml

```diff
@@ -105,6 +105,14 @@
                 {{ package_manager }} remove -y {{ package_manage_list_str }}
               {%- endif -%}
 
+        - name: "Update VMware Photon OS repo cache"
+          include_tasks: repo_update.yml
+          vars:
+            check_update_cmd: "tdnf makecache"
+          when:
+            - guest_os_ansible_distribution == "VMware Photon OS"
+            - package_state in ['installed', 'latest', 'present']
+
         - name: "{{ package_manage_op }} packages on VMware Photon OS"
```

Comment:
Are these tasks executed on both distributions in when condition`when: guest_os_ansible_distribution in ["VMware Photon OS", "FreeBSD"]`, then the task names need to be updated.

Reply:
You are correct. Fixed them.